tools/local/src/docker.ts

```diff
@@ -16,7 +16,13 @@ export async function startContainers(services: Array<string>) {
 
 async function run(cmd: string, opts: { cwd: string }) {
   await new Promise((resolve, reject) => {
-    const p = exec(cmd, opts);
+    const p = exec(cmd, opts, (error, stdout) => {
+      if (error) {
+        // biome-ignore lint/suspicious/noConsoleLog: logging seems the best option here
+        console.log(stdout);
```

Comment:
it's an error, why don't we use `console.error`?

Reply:
Yes, that should have been used